### PR TITLE
[ALLUXIO-2119] Remove use of whitebox in TtlBucketListTest

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-2119

@gpang was it intentional that we don't restore `creationTimeMs` from the journal? Do you think it would make sense to journal Inode's `lastModificationTime`?

### CODE REVIEW

**Comment by jsimsa** [comment]
LGTM

**Comment by aaudiber** [comment]
@gpang was it intentional that we don't restore creationTimeMs from the journal? Do you think it would make sense to journal Inode's lastModificationTime?

**Comment by ifcharming** [comment]
LGTM

File: core/server/src/main/java/alluxio/master/file/meta/InodeFile.java [hunk 1]
```diff
@@ -283,6 +283,7 @@ public static InodeFile fromJournalEntry(InodeFileEntry entry) {
         .setBlockSizeBytes(entry.getBlockSizeBytes())
         .setCacheable(entry.getCacheable())
         .setCompleted(entry.getCompleted())
+        .setCreationTimeMs(entry.getCreationTimeMs())
```
Comment (gpang) [inline core/server/src/main/java/alluxio/master/file/meta/InodeFile.java]: I think technically, the constructor will already set the creation time

File: core/server/src/main/java/alluxio/master/file/meta/InodeFile.java [hunk 2]
```diff
@@ -328,6 +331,7 @@ public JournalEntry toJournalEntry() {
         .setLength(getLength())
         .setCompleted(isCompleted())
         .setCacheable(isCacheable())
+        .setCreationTimeMs(getCreationTimeMs())
```
Comment (gpang) [inline core/server/src/main/java/alluxio/master/file/meta/InodeFile.java]: already on line 323?

**Comment by gpang** [comment]
@aaudiber Thanks! I think the confusion is that we have inode constructors and the `create()` static method, and it is unclear when to use either. There is an inode constructor that takes in the creation time.

File: core/server/src/main/java/alluxio/master/file/meta/InodeFile.java (hunk 1 above)
Comment (aaudiber) [inline core/server/src/main/java/alluxio/master/file/meta/InodeFile.java]: changed constructor to not take creation time

File: core/server/src/main/java/alluxio/master/file/meta/InodeFile.java (hunk 2 above)
Comment (aaudiber) [inline core/server/src/main/java/alluxio/master/file/meta/InodeFile.java]: fixed and alphabetized

**Comment by aaudiber** [comment]
@gpang Thanks for the comments, I didn't realize that the current time could be set via constructor because the field was in the superclass but being set by the subclass. I've reorganized things a bit so that fields are set in alphabetical order and Inode/InodeFile set only their own fields in their constructors.

File: core/server/src/main/java/alluxio/master/file/meta/InodeDirectory.java [hunk 3]
```diff
@@ -55,7 +54,7 @@ public Object getFieldValue(Inode<?> o) {
    * @param id the id to use
    */
   private InodeDirectory(long id) {
-    super(id);
+    super(id, true);
     mDirectory = true;
```
Comment (gpang) [inline core/server/src/main/java/alluxio/master/file/meta/InodeDirectory.java]: this is no longer necessary, right?

**Comment by gpang** [comment]
Thanks @aaudiber . I just had one last minor question, and it looks good.

File: core/server/src/main/java/alluxio/master/file/meta/InodeDirectory.java (hunk 3 above)
Comment (aaudiber) [inline core/server/src/main/java/alluxio/master/file/meta/InodeDirectory.java]: removed, thanks

**Comment by aaudiber** [comment]
@gpang comment addressed, PTAL

**Comment by gpang** [comment]
LGTM

**Pull request merged**